packages/core/database/src/entity-manager/index.ts

```diff
@@ -192,7 +192,7 @@ const processData = (
           : data[joinColumnName];
 
         if (!isUndefined(attrValue)) {
-          obj[joinColumnName] = attrValue;
+          obj[joinColumnName] = toId(attrValue);
```

Comment:
this change allows connecting a join table using a long hand format, which was not supported before